src/DependencyInjection/Configuration.php

```diff
@@ -49,6 +49,11 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->booleanNode('minify')
                             ->defaultFalse()
                         ->end()
+                        ->integerNode('mjml_version')
```

Comment:
Alternatively this could be an enum and rely on consts, but opening a private const is up to maintainers :)